ReactQt/runtime/src/qml/ReactText.qml

```diff
@@ -65,6 +65,20 @@ Text {
     onParentChanged: updateHtmlText()
 
     function updateMeasureFunction() {
+
+        //Only topmost text item in a set of nested ones can have a flexbox node.
+        if(textIsTopInBlock) {
+            textRoot.flexbox = Qt.createQmlObject('import React 0.1 as React; React.Flexbox {control: textRoot}',
```

Comment:
If I'm not wrong, might be less confusing to put `parent` instead of `textRoot` inside new QML object code.

Reply:
@MaxRis, I checked, that doesn't work because Flexbox is not a QQuickItem, it is a QObject* and it doesn't have `parent` property.

Reply:
@vkjr thank you!